manifest.json

```diff
@@ -44,5 +44,13 @@
   "icons": {
     "48": "/assets/icons/48.png",
     "128": "/assets/icons/128.png"
+  },
+
+  "browser_specific_settings": {
```

Comment:
Have you also tested that this does not break the chrome version?

Reply:
Hey!

Yes; after any edit, I always verify that it didn't break something on the Chromium side (either on Chrome or on Vivaldi).

The only side-effect is this:
* When installing the ***unpacked*** extension on Chrome ***while*** having Developer Mode enabled on Chrome, Chrome will display a warning to the developer: `Unrecognized manifest key 'browser_specific_settings'.`

The error does not show up when the developer mode is not enabled. Other extensions that use the same MV3 file for both browsers also seem to be ignoring the warning, e.g. [disboard autobump](https://github.com/Theblockbuster1/disboard-auto-bump)